src/serac/physics/functional_residual.hpp

```diff
@@ -99,13 +100,16 @@ class FunctionalResidual<spatial_dim, OutputSpace, Parameters<InputSpaces...>,
   template <int... active_parameters, typename BodyIntegralType>
   void addBodyIntegral(DependsOn<active_parameters...>, std::string body_name, BodyIntegralType body_integral)
   {
-    residual_->AddDomainIntegral(Dimension<spatial_dim>{}, DependsOn<active_parameters...>{}, body_integral,
-                                 mesh_->domain(body_name));
+    residual_->AddDomainIntegral(
```

Comment:
I'd put another comment in here to future maintainers & modifiers:

```suggestion
    // The body force is negated here so that the residual (internal forces - external forces), and thus its
    // derivative is the stiffness matrix.
    // Applying the negative sign here makes the interface easier, so that users don't have to return
    // negative body forces in body_integral.
    residual_->AddDomainIntegral(
```